### PR TITLE
CategoryLoggerConfig JSON Schema

### PR DESCRIPTION
**Pull request by isaacjwilliams**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,6 +25,9 @@
     "bunyan": "^1.8.15"
   },
   "devDependencies": {
+    "@types/jest": "^28.1.8",
```
Comment (isaacjwilliams) [inline package.json:28]: Useful for tooling (E.G. IntelliJ IDEA).

**Comment by isaacjwilliams** [comment]
I am planning to add documentation in a follow-up PR.

**Review by mknoedel — APPROVED** [review]

File: test/schema.test.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
+const Ajv = require('ajv-draft-04');
+
+const ajv = new Ajv();
+const configSchema = require('../src/config-schema.json');
+const validate = ajv.compile(configSchema);
+
+describe("valid configs", () => {
+  const validConfigs = [
```
Comment (softwarewright) [inline test/schema.test.js:8]: This could just be a test.each

**Review by softwarewright — APPROVED** [review]
I'm guessing you're going to add using the validation later as well.

**Comment by isaacjwilliams** [comment]
@softwarewright This project does not need to use validation. The JSON schema will be used by AppConfig when we update the logging configuration in AWS.

File: test/schema.test.js (hunk 2 above)
Comment (isaacjwilliams) [reply test/schema.test.js:8]: Oops 😬

I forgor 💀

**Pull request merged**